pkg/controllers/controller.go

```diff
@@ -88,6 +91,16 @@ func InstallControllers(stopCh <-chan struct{}, aggregatorConfig *aggregatorapis
 		mgr, err := ctrl.NewManager(loopbackRestConfig, ctrl.Options{
 			Scheme:             scheme,
 			MetricsBindAddress: "0", //TODO think about the mertics later
+			NewCache: cache.BuilderWithOptions(cache.Options{
+				DefaultTransform: func(obj interface{}) (interface{}, error) {
+					k8sObj, ok := obj.(client.Object)
+					if !ok {
+						return nil, fmt.Errorf("invalid type")
+					}
+					k8sObj.SetManagedFields(nil)
```

Comment:
maybe need some comments to explain why we do this

Reply:
done.